roboteam_ai/src/stp/skills/OrbitAngular.cpp

```diff
@@ -1,66 +1,40 @@
-//
-// Created by tijmen on 01-07-22.
-//
-
 #include "stp/skills/OrbitAngular.h"
 
 #include "stp/constants/ControlConstants.h"
+#include "roboteam_utils/Print.h"
 
 namespace rtt::ai::stp::skill {
 
 Status OrbitAngular::onUpdate(const StpInfo &info) noexcept {
-    double rpm = 20;  // Number of orbits (rotations) per minute
-    double circumference = (control_constants::ROBOT_RADIUS + stp::control_constants::BALL_RADIUS) * 2 * M_PI;
-
-    // Constants used to tweak the orbit to be a nice circle
-    constexpr double ANGLE_RATE_FACTOR = 1.0;
-    constexpr double VELOCITY_FACTOR = 1.2;
-
-    // Calculate angVel and rotational velocity to make a circular movement around the ball
-    double angVel = -rpm / 60 * 2 * M_PI * ANGLE_RATE_FACTOR;
-    double rotVel = rpm / 60 * circumference * VELOCITY_FACTOR;
-
-    Vector2 directionVector = info.getRobot()->get()->getAngle().toVector2();
-    Angle targetAngle = (info.getPositionToShootAt().value() - info.getRobot()->get()->getPos()).toAngle();
+    RTT_DEBUG(info.getBall().value()->velocity.length())
```

Comment:
Perhaps it is better to not leave this print statement in the code. If these print statements happen more often, eventually the terminal will get cluttered by so many statements that the few important statements are difficult to notice.

If you do want to keep this statement in here, I suggest adding some text around it so when it prints, on the same line it explains what the value means, not just a random number like 1.591 :)

Reply:
It was indeed not the idea to leave this in the code. Will get rid of it.